models/GroupsService.php

```diff
@@ -74,48 +95,47 @@ public function deleteGroup(\core_kernel_classes_Resource $group)
      *
      * @access public
      * @author Joel Bout, <joel.bout@tudor.lu>
-     * @param  Class clazz
+     * @param  core_kernel_classes_Class $clazz
      * @return boolean
      */
     public function isGroupClass(core_kernel_classes_Class $clazz)
     {
-        return $clazz->equals($this->getRootClass()) || $clazz->isSubClassOf($this->getRootClass());
+        return $clazz->equals($this->getRootClass())
+            || $clazz->isSubClassOf($this->getRootClass());
     }
 
     /**
-     * get the groups of a user
+     * Get the groups of a user
      *
      * @access public
      * @author Joel Bout, <joel.bout@tudor.lu>
-     * @param  string userUri
+     * @param  User $user
```

Comment:
We can remove this annotation :)

Reply:
Removing annotations with type hints and adding missing ones in df4395d5c3ee253c6b16d1b20b28d51b512b460d & 6be2c8e9e923b55fc870cb31c59732aa1a5b4df9 , thanks.